mailmerge.py

```diff
@@ -136,36 +136,110 @@ def get_merge_fields(self, parts=None):
                 fields.add(mf.attrib['name'])
         return fields
 
-    def merge_pages(self, replacements):
+    def merge_templates(self, replacements, separator, type):
         """
-        Duplicate template page. Creates a copy of the template for each item
-        in the list, does a merge, and separates the them by page breaks.
+        Duplicate template. Creates a copy of the template, does a merge, and separates the them by a break or a new section break (separator argument : 'break' or 'section').
+        the 'type' argument determines witch type of break or new section is inserted :
+
+        For Break, types can be :
```

Comment:
I think it would be nice to combine separator and type into a single `Separator` enum, something like this:

```python
@unique
class Separator(enum):
    page_break = ...
    column_break
    text_wrapping
    continuous_section
    even_page_section
    next_column_section
    next_page_section
    odd_page_section
```

That would resolve the issue of providing incorrect combinations and is also explicit about the accepted break types.